src/languages/ko-KR.js

```diff
@@ -520,12 +520,62 @@ export default {
     "Steam Game Server": "스팀 게임 서버",
-    "Most likely causes:": "원인:",
-    "The resource is no longer available.": "더 이상 사용할 수 없어요...",
-    "There might be a typing error in the address.": "주소에 오탈자가 있을 수 있어요.",
+    "Most likely causes:": "가능성 높은 원인:",
+    "The resource is no longer available.": "존재하지 않는 자료입니다.",
+    "There might be a typing error in the address.": "주소에 오탈자가 있을 수 있습니다.",
     "What you can try:": "해결 방법:",
     "Retype the address.": "주소 다시 입력하기",
     "Go back to the previous page.": "이전 페이지로 돌아가기",
-    "Coming Soon": "Coming Soon...",
-    wayToGetClickSendSMSToken: "{0}에서 API 사용자 이름과 키를 얻을 수 있어요.",
+    "Coming Soon": "곧 찾아옵니다...",
+    wayToGetClickSendSMSToken: "{0}에서 API 사용자 이름과 키를 얻을 수 있습니다.",
+    resendEveryXTimes: "매 {0}회 마다 다시 전송",
+    resendDisabled: "재전송 비활성화됨",
+    "Heartbeat Timeout": "하트비트 타임아웃",
+    timeoutAfterMs: "{0}ms 후 타임아웃",
+    "Resend Notification if Down X times consequently": "X회 연속 다운된 경우 알림 재전송",
+    "Bark Group": "Bark 그룹",
+    "Bark Sound": "Bark 소리",
+    "HTTP Headers": "HTTP 헤더",
+    "Trust Proxy": "프록시 신뢰",
+    HomeAssistant: "홈 어시스턴트",
+    RadiusSecret: "Radius 비밀 정보",
+    RadiusSecretDescription: "클라이언트와 서버간 공유 비밀 키",
+    RadiusCalledStationId: "호출된 스테이션 아이디",
+    RadiusCalledStationIdDescription: "호출된 기기 식별자",
+    RadiusCallingStationId: "호출한 스테이션 아이디",
+    RadiusCallingStationIdDescription: "호출하는 기기 식별자",
+    "Connection String": "연결 문자열",
+    Query: "쿼리",
+    settingsCertificateExpiry: "TLS 인증서 만료",
+    certificationExpiryDescription: "TLS 인증서가 다음 안에 만료되는 경우 HTTPS 모니터가 알림을 보냅니다:",
+    "Setup Docker Host": "Docker 호스트 설정",
+    "Connection Type": "연결 형태",
+    "Docker Daemon": "Docker 데몬",
+    deleteDockerHostMsg: "모든 모니터에서 이 Docker 호스트를 삭제하시겠습니까?",
+    socket: "소켓",
+    tcp: "TCP / HTTP",
+    "Docker Container": "Docker 컨테이너",
+    "Container Name / ID": "컨테이너명 / ID",
+    "Docker Host": "Docker 호스트",
+    "Docker Hosts": "Docker 호스트들",
+    "ntfy Topic": "ntfy 토픽",
+    Domain: "도메인",
+    Workstation: "워크스테이션",
+    disableCloudflaredNoAuthMsg: "현재 비인증 모드이므로, 비밀번호가 필요하지 않습니다.",
+    wayToGetLineNotifyToken: "{0}에서 액세스 토큰을 얻을 수 있습니다.",
+    Examples: "예제",
+    "Home Assistant URL": "홈 어시스턴트 URL",
+    "Long-Lived Access Token": "장기 액세스 토큰",
+    "Notification Service": "알림 서비스",
+    "default: notify all devices": "기본: 모든 기기에 알림",
+    "Automations can optionally be triggered in Home Assistant:": "자동화를 홈 어시스턴트에서 선택적으로 동작시킬 수 있습니다.:",
+    "Trigger type:": "트리거 유형:",
+    "Event type:": "이벤트 유형:",
+    "Event data:": "이벤트 데이터:",
+    "Then choose an action, for example switch the scene to where an RGB light is red.": "이후 액션을 선택하세요, 예를 들어 RGB 불빛이 적색인 Scene 등으로 교체합니다.",
+    "Frontend Version": "프론트엔드 버전",
+    "Frontend Version do not match backend version!": "프론트엔드 버전이 백엔드 버전과 맞지 않습니다!",
+    "Base URL": "기본 URL",
+    goAlert: "GoAlert",
+    backupRecommend: "볼륨 또는 데이터 폴더 (./data/) 를 대신 직접 백업해주세요.",
```

Comment:
Hello team, I've reverted every un-touched languages change. Please take a look again and see if there's something I could improve. In the meantime I may take a look into the script itself too but I'll refrain myself from mixing 2 features into 1 PR 😄